src/get-jwks.js

```diff
@@ -4,63 +4,66 @@ const fetch = require('node-fetch')
 const lru = require('tiny-lru')
 const jwkToPem = require('jwk-to-pem')
 
-const MISSING_KEY_ERROR = 'No matching key found in the set.'
-const NO_KEYS_ERROR = 'No keys found in the set.'
+const MISSING_JWK_ERROR = 'No matching JWK found in the set.'
+const NO_JWKS_ERROR = 'No JWKS found in the set.'
 
 function buildGetJwks (cacheProps = {}) {
   const max = cacheProps.max || 100
   const ttl = cacheProps.ttl || 60 * 1000
   const cache = lru(max, ttl)
 
-  async function getSecret (signatures) {
+  async function getPublicKey (signatures) {
     const { domain, alg, kid } = signatures
     const cacheKey = `${alg}:${kid}:${domain}`
-    const cachedSecret = cache.get(cacheKey)
+    const cachedJwk = cache.get(cacheKey)
```

Comment:
all this function should do is call getJwk and convert the result to PEM

Reply:
Ah I see, yep thanks!